packages/typescript-resolver-files-e2e/src/test-resolvers-auto-wireup/schema/base/resolvers/CustomLogicScalar.ts

```diff
@@ -1,3 +1,4 @@
-import { DateResolver } from 'graphql-scalars';
-DateResolver.description = undefined;
-export const CustomLogicScalar = DateResolver;
+import { DateResolver} from 'graphql-scalars'
+      DateResolver.description = undefined;
+      export const CustomLogicScalar = DateResolver;
+      
```

Comment:
These files are changed because:
- What we are seeing here is the un-formatted version by the setup
- It means the files are not "tracked" by codegen and not run prettier on because of codegen's hooks
- This is expected